site/pt.njk

```diff
@@ -127,13 +127,17 @@ lang: pt
+                <br />
+                Se tem algum outro tipo de contribuição relacionada com o conteúdo do website, por favor escreva-nos para <a href="mailto:contribute@staythefuckhome.com">contribute@staythefuckhome.com</a>.
+                Note que actualmente estamos a receber imensos emails, pelo que podemos não lhe responder directamente.
+            </p>
 
             <p class="text-sm text-gray-500 pt-12 max-w-4xl mx-auto">
-                <strong>Disclaimer:</strong> The content of this website does not constitute medical advice. The information provided above is meant to be a suggestion for concerned individuals to follow a loose set of guidelines
-                based on best practices and anecdotal evidence to protect themselves and their loved ones. There is no guarantee or scientific evidence that following the mentioned guidelines will lead to a reduction
-                in new infections or even stop the COVID-19 epidemic. There is also no guarantee that following these guidelines will reduce the risk of getting infected with SARS-CoV-2 or any other viral or bacterial contagion.
-                All information provided on this website is given with the best intent and will to provide factual information. In no event shall the website operators be held liable for any claim, damages or other liabilities. If you have concerns or comments about the information provided on this website,
-                please write to <a class="text-gray-600 hover:text-pink-500" href="mailto:inquiry@staythefuckhome.com">inquiry@staythefuckhome.com</a>.
+                 <strong>Aviso legal:</strong> O conteúdo deste website não constitui conselho médico. A informação disponibilizada acima tenciona ser uma sugestão para indivíduos interessados seguirem um conjunto de linhas orientadoras baseadas nas melhores práticas e evidências informais para se protegerem a si e aos seus. Não há garantia ou evidência científica que as referidas linhas orientadoras levarão a uma redução no número de infecções ou que travem a pandemia da COVID-19. Também não há garantia que seguir estas linhas orientadoras reduza o risco de ser infectado com o SARS-CoV-2 ou qualquer outro contágio viral ou bacteriano.
+                 Toda a informação fornecida neste website é disponibilizada com a melhor intenção e não deverá servir como informação factual. Em caso algum serão os operadores deste website responsabilizados por qualquer reivindicação, danos ou outras obrigações. Se tem alguma preocupação ou comentário a fazer sobre a informação fornecida neste website, por favor escreva para <a class="text-gray-600 hover:text-pink-500" href="mailto:inquiry@staythefuckhome.com">inquiry@staythefuckhome.com</a>.
```

Comment:
If I'm not mistaken, there is a slight error in the translation here. The English reads `and will to provide factual information.` Your translation, if I'm not mistaken, says that website should not be taken as factual information, which is quite the opposite. Can you revise this?

Reply:
Indeed there is, well caught! Fixing it right away.

Reply:
Done.